recipe/meta.yaml

```diff
@@ -33,8 +33,7 @@ requirements:
     - python-dateutil
     - myproxyclient
     - esgf-pyclient
-    # Will need this later for the 1.2.0 release, leaving commended for now
-    # - podaacpy >= 1.4.0
+    - podaacpy
 
 test:
   imports:
```

Comment:
Uncomment out this line: `license_file: LICENSE.txt`, and remove the two comment lines above it.